### PR TITLE
unit test show qrcode

### PR DESCRIPTION
**Pull request by ghost**
## Description
Unit test show qrcode book
## Reference Requirements
- Product Backlog ID: PBI-8
- Task ID: #8.2

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
Non SQL

### CODE REVIEW

File: tests/Browser/Pages/Backend/Books/AdminListBooksTest.php [hunk 1]
```diff
@@ -80,6 +80,28 @@ public function testListBooksHasRecord()
...
+    /**
+     * Test view Admin List Books if database has record column qrcode
+     *
+     * @return void
+     */
+    public function testListBooksHasRecordColumnQrcode()
+    {
+        $this->makeData(5);
+        $qrcode = $this->makeQueryQrcode()->toArray();
+        $this->browse(function (Browser $browser) use($qrcode) {
```
Comment (at-caonguyen) [inline tests/Browser/Pages/Backend/Books/AdminListBooksTest.php]: thiếu 1 space sau use nhé

**Review by mvn-caonguyen-dn — COMMENTED** [review]
check cmt

**Comment by at-caonguyen** [comment]
LGTM :+1:

File: tests/Browser/Pages/Backend/Books/AdminListBooksTest.php [hunk 2]
```diff
@@ -80,6 +80,28 @@ public function testListBooksHasRecord()
         });
     }
 
+    /**
+     * Test view Admin List Books if database has record column qrcode
```
Comment (at-huydq) [inline tests/Browser/Pages/Backend/Books/AdminListBooksTest.php]: Test thêm trường hợp ko có Qrcode luôn nha

**Comment by at-huydq** [comment]
LGTM.

**Pull request merged**